source/voice_control/s3-box-customize.markdown

```diff
@@ -0,0 +1,189 @@
+
+1. Make sure the ESP32-S3-BOX is up and running and connected to your Wi-Fi.
+2. Go to **Settings** > **Add-ons** and check if you have the **ESPHome** add-on installed.
+   - If you haven't done so, install the add-on.
+3. Start the add-on.
+4. Once the add-on is running, select **Open web UI**.
+5. In the ESPHome add-on dashboard, on the **ESP32-S3-BOX** card, select **Adopt**.
+   ![Adopt the ESP32-S3-BOX in the ESPHome add-on](/images/assist/esp32-adopt-s3.png)
+6. If you like, give it a new name. Then, select **Adopt**.
+7. Copy the encryption key and store it in a safe place.
+   - It will also be available within the UI, but it is always good to keep a copy.
+   - **Result**: Once you copied the key, the status will change to **Online**.
+8. Now that you have set up the ESPHome add-on, continue with one of the 2 different methods to add custom images:
+   - [Using images from a community repository](#using-images-from-a-community-repository)
+   - [Using your own images](#using-your-own-images)
+
+## Using images from a community repository
+
+- Perform this procedure, if you want to use ready-made images by the community.
+- If you want to use your own images, skip this procedure and go to [Using your own images](#using-your-own-images) instead.
+
+**Duration**: about 30 minutes
+
+### To use images from the community
+
+1. On the **ESP32-S3-BOX** add-on, select edit.
+   - **Result**: An editor opens, showing the configuration file.
+   ![ESP32-S3-BOX config file](/images/assist/esp32-adopt-s3-01.png)
+2. For inspiration, we have prepared some images for you.
+   - Check them out in this [repository](https://github.com/jlpouffier/home-assistant-s3-box-community-illustrations/tree/main).
+3. For this tutorial, we will use some images of Frenck.
+   - Add the following lines into the `substitution` block.
+
+   {% raw %}
+
+     ```yaml
+      loading_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/loading_320_240.png
+      idle_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/idle_320_240.png
+      listening_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/listening_320_240.png
+      thinking_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/thinking_320_240.png
+      replying_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/replying_320_240.png
+      error_illustration_file: https://raw.githubusercontent.com/jlpouffier/home-assistant-s3-box-community-illustrations/main/frenck/illustrations/error_320_240.png
+    ```
+
+   {% endraw %}
```

Comment:
![CleanShot 2023-12-12 at 12 43 01](https://github.com/home-assistant/home-assistant.io/assets/1444314/69ae8067-a843-41da-bf4d-1e9595f60b1f)
I don't know where these 2 artifacts come from on the page, maybe because of the extra new lines here? Why are we using `raw` tags ?